crates/sui-core/src/authority.rs

```diff
@@ -3460,6 +3463,13 @@ impl AuthorityState {
             .pending_notify_read
             .set(self.database.executed_effects_notify_read.num_pending() as i64);
 
+        // count signature by scheme, for zklogin and multisig
+        if certificate.has_zklogin_sig() {
+            self.metrics.signature_metrics.zklogin_sig_count.inc();
+        } else if certificate.has_upgraded_multisig() {
+            self.metrics.signature_metrics.multisig_sig_count.inc();
+        }
```

Comment:
i think it might make more sense to do this in handle_certificate or handle_transaction

@lxfind what do you think?

Reply:
I don't have strong opinions here. We are not very disciplined on metrics.